xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variables.less

```diff
@@ -5,6 +5,10 @@
 @figure-caption-color:              @text-muted;
 @loginFormMaxWidth:                 450px;
 
+// Standard pointer target sizes from WCAG recommendations
+@target-size-minimum:                 24px;
+@target-size-recommended:             44px;
```

Comment:
The alignment of the values seems slightly off (I guess the `24/44px` should be aligned with the `450px` above?)